packages/storybook/templates/cxl-vaadin-accordion.js

```diff
@@ -0,0 +1,78 @@
+import { html } from 'lit-html';
+
+export default html`
+  <cxl-vaadin-accordion>
+    <vaadin-accordion-panel theme="cxl-sidebar-panel">
```

Comment:
@chanar there is no matching theme style module. Is this attribute an error?

Reply:
`<vaadin-accordion-panel>` uses vaadin-details therefore styling must be done through vaadin-details template

Reply:
Also, it adds theme set for vaadin-accordion-panel to vaadin-details

Reply:
Looks like this explains theme attribute propagation a bit https://vaadin.com/forum/thread/17596415/

But in the markup below, there is no `<vaadin-details>` element, it's been refactored out.

Another vote for refactoring the current untrackable styling approach. I'm working on it right now.

Reply:
Mkay, new learnings: `class AccordionPanelElement extends Vaadin.DetailsElement`

Reply:
In refactor, I went with `theme-for="vaadin-accordion-panel"` beacuse that's the actual element, and it seems to work fine for now.